react/README.md

```diff
@@ -236,10 +237,34 @@
     });
     ```
 
+## Arrow Functions
+  - When using ES2015 syntax and declaring your components as `class MyComponent extends React.Components {}` you should use arrow functions to define functions that will need to be bound to the components scope. This removes the need to have `.bind()` ran on every render. The arrow function will have the lexical `this` of the component.
+  ```javascript
+  // bad
+  onClick() {
+    // do stuff
+  }
+
+  render() {
+    return <a onClick={this.onClick.bind(this)}>Clicky</a>;
+  }
+
+  // good
+  onClick = () => {
+    // do stuff
+  }
+
+  render() {
+    return <a onClick={this.onClick}>Clickier</a>;
+  }
```

Comment:
I thought about raping them in Component definitions. Was gonna see if anyone thought it would be cool.